core/arch/arm/mm/core_mmu_lpae.c

```diff
@@ -726,7 +730,6 @@ static void core_init_mmu_prtn_ta_core(struct mmu_partition *prtn
 		assert((tbl[0] & DESC_ENTRY_TYPE_MASK) == TABLE_DESC);
 
 		tbl = core_mmu_xlat_table_entry_pa2va(prtn, 0, tbl[0]);
-
```

Comment:
(nit) this change belongs to commit "core: lpae: support user mapping when base level is 0".